spec/controllers/settings_controller_spec.rb

```diff
@@ -1,32 +1,41 @@
-require 'spec_helper'
-
-describe SessionsController do
+ require 'spec_helper'
+
+describe SettingsController, type: :controller do
+   
+  let(:user) { FactoryGirl.create(:user) }
+=begin
```

Comment:
Done

On Thu, Apr 9, 2015 at 1:17 PM, Ben Woodall <notifications@github.com>
wrote:

> In spec/controllers/settings_controller_spec.rb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28098434>:
>
> > @@ -1,32 +1,41 @@
> > -require 'spec_helper'
> > -
> > -describe SessionsController do
> > + require 'spec_helper'
> > +
> > +describe SettingsController, type: :controller do
> > +
> > +  let(:user) { FactoryGirl.create(:user) }
> > +=begin
>
> Clean up commented code!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28098434>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw